requirements.txt

```diff
@@ -3,7 +3,7 @@ openstacksdk
 # for various "side-effects" such as creating "default" security
 # groups....don't ask...
 python-openstackclient == 6.3.0
-https://github.com/stfc/openstack-query-library/releases/download/v1.3.0/openstackquery-1.3.0-py3-none-any.whl
+https://github.com/stfc/openstack-query-library/releases/download/v1.1.0/openstackquery-1.1.0-py3-none-any.whl
```

Comment:
Why has the minor version changed to 1.1 here?

Reply:
v1.2 added the changed the API here while introducing the broken changes: https://github.com/stfc/openstack-query-library/releases/tag/v1.2.0 in https://github.com/stfc/openstack-query-library/pull/22
(It should have been a 2.0.0 release for changing the public API, but can't change that now)

This reverts to the first release prior to these changes